api/src/test/java/uk/gov/dhsc/htbhf/claimant/message/MessageProcessorTest.java

```diff
@@ -119,6 +124,31 @@ void shouldContinueToProcessMessagesAfterAnExceptionThrown() {
         verify(messageStatusProcessor).processStatusForMessage(cardMessage2, FAILED);
         verify(messageStatusProcessor).processStatusForMessage(cardMessage3, COMPLETED);
         verifyNoMoreInteractions(messageRepository, messageStatusProcessor, createNewCardDummyMessageTypeProcessor);
+        verifyZeroInteractions(eventAuditor);
+    }
+
+    @Test
+    void shouldAuditFailedEventAnEventFailedFailedExceptionThrown() {
```

Comment:
This might need renaming :)

Reply:
Um, yes, well, how embarrassing, all fixed. :-)